addon/adapters/rest.js

```diff
@@ -104,6 +124,24 @@ const {
   }
   ```
 
+  ### Errors
+
+  If a response is considered a failure, the JSON payload is expected to include
+  a top-level key `errors`, detailing any specific issues. For example:
+
+  ```js
+  {
+    "errors": {
+      "msg": "Something went wrong"
+    }
+  }
+  ```
+
+  This adapter does not make any assumptions as to the format of the `errors`
+  object. It will simply be passed along as a black box, wrapped in an instance
```

Comment:
Hmm, IMHO "be passed along as a black box, wrapped ..." is a little confusing here. What do you think of "be passed along as is, wrapped ..."?